tools/check-commit-messages/check_commit_messages.py

```diff
@@ -42,8 +42,14 @@ def check_commit_message(commit):
     if not body_lines:
         return False, f"Commit {commit.hexsha} has no body."
 
+    skip_capital_letter_check = False
```

Comment:
I'd suggest that this should be false for all files.
FWIW I just had a commit with a body which started with a filename
"device-steps.sh waits ..." which doesn't make sense to capitalize!

Also, in the new format the body is required - means that simple commits e.g., with a title like "Update vendor ..." will need a body.

Reply:
I think requiring a body is fine since it forces contributors to think what there is to explain.